visitor_actions.admin.inc

```diff
@@ -272,17 +275,6 @@ function visitor_actions_form_validate($form, &$form_state) {
         form_set_error('identifier[' . $plugin . ']', t('The selector cannot contain tags.'));
       }
     }
-
```

Comment:
Removed the following validation in favor of using visitor_actions_form_element_path_validate() against just the the 'page' field.